tests/test_utils.py

```diff
@@ -0,0 +1,26 @@
+import sys
+sys.path.append('../')
```

Comment:
Remove these two lines, they are not needed if you run the tests using `python2 -m unittest tests.test_utils` or `python3 -m unittest tests/test_utils.py`.